draft-ietf-ppm-dap.md

```diff
@@ -160,10 +160,23 @@ aggregator.
 
 15:
 
-- Reorganize text for clarity and flow.
+- Specify body of responses to aggregation job GET requests. (#651)
+
+- Add diagram illustrating object lifecycles and relationships. (#655)
+
+- Use aasvg for prettier diagrams. (#657)
+
+- More precise description of time and intervals. (#658)
+
+- Reorganize text for clarity and flow. (#659, #660, #661, #663, #665, #666,
+  #668, #672, #678, #680, #684, #653, #654)
+
+- Align with RFC 9205 recommendations. (#673, #683)
```

Comment:
Is this a breaking change?

Reply:
Not as far as I can tell.

Reply:
In my view it is. Previously we were prescribing specific HTTP status codes. Now we're not, but a DAP-14 aggregator might expect us to.